internal/providers/terraform/aws/eks_fargate_profile.go

```diff
@@ -0,0 +1,68 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetNewEKSFargateProfileItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_eks_fargate_profile",
```

Comment:
I'm not seeing anything obvious in `aws_eks_*` terraform resources where a user can specify the CPU/RAM of the fargate containers, whereas for `aws_ecs_*` fargate resources there is memory/cpu in https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition (used by the `convertResourceString` method in `ecs_service.go`). Any idea how that works?

Reply:
I was under the assumption that it will be managed by the kube config?